packages/gatsby-plugin-react-helmet/README.md

```diff
@@ -26,4 +26,3 @@ plugins: [`gatsby-plugin-react-helmet`]
 ## Examples
 
 - [GatsbyJS.org](https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/site-metadata.js)
-- [Jason Lengstorf personal website](https://github.com/jlengstorf/lengstorf.com/blob/master/src/components/SEO/SEO.js)
```

Comment:
The SEO component was moved to https://github.com/jlengstorf/gatsby-theme-jason-blog/blob/master/src/components/SEO/SEO.js